tests/UI/pages/BO/catalog/attributes/index.js

```diff
@@ -363,6 +367,39 @@ class Attributes extends BOBasePage {
     }
     return allRowsContentTable;
   }
+
+  // Help card methods
+  /**
+   * @override
```

Comment:
Why is this an override ? The helpcard system is different on this page ?

Reply:
since it exist for all pages, it's BOBasePage function. the selectors are not the same as in legacy pages (that's why the override)